config/rukpak/core/resources/deployment.yaml

```diff
@@ -65,6 +70,10 @@ spec:
             - "--bundle-ca-file=/etc/pki/tls/tls.crt"
           ports:
             - containerPort: 8080
+          resources:
+            requests:
+              cpu: 15m
+              memory: 100Mi
```

Comment:
This is for the core rukpak deployment. I increased the memory request to 100Mi just because we're caching bundle contents in-memory, so we may want to invest in benchmarking this behavior to get a better read on what kind of requests we need to be setting for subsequent phases.